ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Xcode11/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Xcode 11.7 is needed to build x64 JDK8 om Arm64 Mac
+# See https://github.com/adoptium/infrastructure/issues/2536#issuecomment-1708716478
+# The Xcode11.7 package should be present in /Vendor_Files/mac/Xcode_11.7.xip on the controller node
```

Comment:
Can we add something in here to indicate where the package comes from (i.e. "Log into developer.apple.com with an Apple developer ID to download this package" (I don't know if that URL is correct so please adjust accordingly ;-) ) to make it easy for others to use this role if desired.